tiatoolbox/models/architecture/vanilla.py

```diff
@@ -79,6 +80,45 @@ def _get_architecture(
     return model.features
 
 
+def _get_timm_architecture(
+    arch_name: str,
+    weights: str or WeightsEnum = "DEFAULT",
+    **kwargs: dict,
+) -> list[nn.Sequential, ...] | nn.Sequential:
+    """Get architecture and weights for pathology-specific timm models.
+
+    Args:
+        arch_name (str):
+            Architecture name.
+        weights:
+            path to pretrained weights
+        kwargs (dict):
+            Key-word arguments.
+
+    Returns:
+        A ready-to-use timm model.
+    """
+    if arch_name == "uni_v1":
+        # UNI tile encoder: https://huggingface.co/MahmoodLab/UNI
+        feat_extract = timm.create_model(
+            "hf-hub:MahmoodLab/UNI",
+            pretrained=True,
+            init_values=1e-5,
+            dynamic_img_size=True,
+        )
+    elif arch_name == "prov-gigapath" and timm.__version__ > "1.0.0":
+        # ProViT-GigaPath tile encoder: https://huggingface.co/prov-gigapath/prov-gigapath
+        # There is a bug in timm version 0.9.8 (https://github.com/prov-gigapath/prov-gigapath/issues/2). Version 1.0.3 has been tested successfully.
+        feat_extract = timm.create_model(
+            "hf_hub:prov-gigapath/prov-gigapath", pretrained=True
+        )
+    else:
+        msg = f"Architecture {arch_name} not supported. If you are loading timm models, only timm version > `1.0.3` are supported."
```

Comment:
This line is too long and causing the tests to fail. If you do `pre-commit install` in your development environment then it will test formatting before making a commit. You can also run `ruff` manually in your terminal before making a commit.
```suggestion
        msg = f"Architecture {arch_name} not supported." 
              f"If you are loading timm models, only timm version > `1.0.3` are supported."
```

Reply:
Thanks! I will do it from now on.